lib/send_sms.rb

```diff
@@ -0,0 +1,24 @@
+require "dotenv/load"
+# Download the helper library from https://www.twilio.com/docs/ruby/install
+require "rubygems"
+require "twilio-ruby"
+require "time"
+
+# Your Account Sid and Auth Token from twilio.com/console
+# DANGER! This is insecure. See http://twil.io/secure
+
+class Message
+  def initialize
+  account_sid = ENV["TWILIO_ACCOUNT_SID"]
```

Comment:
The `TwilioClient` could be a dependency on this class so that this could be tested